docs/rules/terraform_blacklisted_resources.md

```diff
@@ -0,0 +1,50 @@
+# terraform_blacklisted_resources
+
+Disallow `resource` declarations of certain types. Each type has a custom message will be displayed.
+
+## Example
+
+```hcl
+rule "terraform_blacklisted_resources" {
+  enabled = true
+  types   = {
+    aws_iam_policy_attachment = "Consider aws_iam_role_policy_attachment, aws_iam_user_policy_attachment, or aws_iam_group_policy_attachment instead."
```

Comment:
What about a `map[string][]string`, where the key is the forbidden resource and the values are the suggested alternatives? I think it's best that we avoid putting literal messages in rule configuration, unless it's a meta-parameter that all rules can provide (https://github.com/terraform-linters/tflint/issues/768).